paymentsheet/src/test/java/com/stripe/android/paymentsheet/flowcontroller/FlowControllerConfigurationHandlerTest.kt

```diff
@@ -118,27 +120,30 @@ class FlowControllerConfigurationHandlerTest {
     @Test
     fun `configure() should re-run CHANGED config during second config`() = runTest {
```

Comment:
Should we have 2 tests? One where a config item changes, and one where an elementssessionparams changes?